runtime/src/qio/qio_formatted.c

```diff
@@ -2231,257 +2331,170 @@ int _ftoa(char* restrict dst, size_t size, double num, int base, bool needs_i, c
     shownegative = isnegative;
   }
 
+  showbase = 0;
+  if( style->prefix_base && base == 16 && !isnan(num) && !isinf(num) ) {
```

Comment:
I don't remember anything about binary/octal - there was a request for hex float literals but I have that under control in another patch... but this code doesn't do anything with literals; just input/output.